src/diffusers/loaders.py

```diff
@@ -219,7 +250,10 @@ def save_attn_procs(
             return
 
         if save_function is None:
-            save_function = torch.save
+            if safe_serialization:
+                save_function = safetensors.torch.save_file
```

Comment:
Can we save the format here as well as this is needed in some loading code? 

E.g. see updated code here: 
https://github.com/huggingface/diffusers/blob/856dad57bb7a9ee13af4a08492e524b0a145a2c5/src/diffusers/models/modeling_utils.py#L311